api/minekube/gate/v1/gate_service.proto

```diff
@@ -191,4 +228,281 @@ message Player {
   string id = 1;
   // The player's username
   string username = 2;
+  // Optional Bedrock player data (only present for Bedrock players)
+  BedrockPlayerData bedrock = 3;
+}
+
+// BedrockPlayerData contains information specific to Bedrock Edition players.
+// This data is only available for players connecting through Geyser/Floodgate.
+message BedrockPlayerData {
+  // Xbox User ID (XUID) - unique identifier for Bedrock players
+  int64 xuid = 1;
+  // Device operating system the player is using
+  string device_os = 2;
+  // Client language code (e.g., "en_US")
+  string language = 3;
+  // UI profile: 0 = Classic, 1 = Pocket
+  int32 ui_profile = 4;
+  // Input mode: 0 = Unknown, 1 = Mouse, 2 = Touch, 3 = GamePad, 4 = MotionController
+  int32 input_mode = 5;
+  // Whether the player is connecting through a proxy
+  bool behind_proxy = 6;
+  // Linked Java Edition username (if any)
+  string linked_player = 7;
+}
+
+// GetStatusRequest is the request for GetStatus method.
+message GetStatusRequest {}
+
+// ProxyMode enumerates the current operating mode of Gate.
+enum ProxyMode {
+  PROXY_MODE_UNSPECIFIED = 0;
+  PROXY_MODE_CLASSIC = 1;
+  PROXY_MODE_LITE = 2;
+}
+
+// GetStatusResponse contains proxy runtime metadata.
+message GetStatusResponse {
+  string version = 1;
+  ProxyMode mode = 2;
+  int32 players = 3;
+  int32 servers = 4;
+}
+
+// ConfigFormat specifies the serialization format for config payloads.
+enum ConfigFormat {
+  CONFIG_FORMAT_UNSPECIFIED = 0;
+  CONFIG_FORMAT_JSON = 1;
+  CONFIG_FORMAT_YAML = 2;
+}
+
+// GateConfig represents the root configuration structure
+message GateConfig {
+  APIConfig api = 1;
+  HealthServiceConfig health_service = 2;
+  ConnectConfig connect = 3;
+  JavaConfig config = 4;
+}
+
+// APIConfig represents the Gate API configuration
+message APIConfig {
+  bool enabled = 1;
+  string bind = 2;
+}
+
+// HealthServiceConfig represents the health service configuration
+message HealthServiceConfig {
+  bool enabled = 1;
+  string bind = 2;
+}
+
+// ConnectConfig represents the Connect network configuration
+message ConnectConfig {
+  bool enabled = 1;
+  string name = 2;
+  bool allow_offline_mode_players = 3;
+}
+
+// JavaConfig represents the main Java edition configuration
+message JavaConfig {
+  string bind = 1;
+  bool online_mode = 2;
+  ForwardingConfig forwarding = 3;
+  StatusConfig status = 4;
+  map<string, string> servers = 5;
+  repeated string try = 6;
+  map<string, StringList> forced_hosts = 7;
+  bool accept_transfers = 8;
+  bool bungee_plugin_channel_enabled = 9;
+  LiteConfig lite = 10;
+  // Add other commonly used fields as needed
+}
+
+// ForwardingConfig represents player info forwarding settings
+message ForwardingConfig {
+  string mode = 1;
+  string velocity_secret = 2;
+  string bungee_guard_secret = 3;
+}
+
+// StatusConfig represents status response settings
+message StatusConfig {
+  string motd = 1;
```

Comment:
I’m missing doc comments above the fields. That goes with all fields in the PR. Yes it’s a lot but you can simply copy them over from the Go code